README.md

```diff
@@ -123,7 +123,9 @@ Make sure to clone this repository in your `GOPATH` - otherwise compilation fail
 
 ```sh
 $ git clone https://github.com/projectatomic/skopeo $GOPATH/src/github.com/projectatomic/skopeo
-$ cd $GOPATH/src/github.com/projectatomic/skopeo && make binary-local
+$ cd $GOPATH/src/github.com/projectatomic/skopeo && 
+Fedora$ make binary-local
```

Comment:
For me uname does return `Darwin`

The test doesn't work though, but I think that is because my shell is zsh... Certainly we could put a test in the Makefile to do this automatically ...